packages/netlify-cms-backend-github/scripts/createFragmentTypes.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+const path = require('path');
+
+const API_HOST = process.env.GITHUB_HOST || 'https://api.github.com';
+const API_TOKEN = process.env.GITHUB_API_TOKEN;
+
+if (!API_TOKEN) {
+  throw new Error('Missing environment variable GITHUB_API_TOKEN');
+}
+
+fetch(`${API_HOST}/graphql`, {
```

Comment:
see https://www.apollographql.com/docs/react/advanced/fragments/#fragments-on-unions-and-interfaces